documentation/docs/developers/connectors/index.md

```diff
@@ -60,7 +60,9 @@ The `execute_buy` and `execute_sell` methods verify that the trades would be leg
 
 ## Requirements for community-developed connectors
 
-<pre><b>If you would like to create a connector for a currently unsupported exchange, please contact the Hummingbot team to discuss beforehand and for approval. Due to the large amount of work in reviewing, testing, and maintaining exchange connectors, we will only merge in connectors that will have a meaningful benefit and impact to the Hummingbot community.</b></pre>
+```
+If you would like to create a connector for a currently unsupported exchange, please contact the Hummingbot team to discuss beforehand and for approval. Due to the large amount of work in reviewing, testing, and maintaining exchange connectors, we will only merge in connectors that will have a meaningful benefit and impact to the Hummingbot community.
```

Comment:
@dennisocana I think for this you can use one of the info boxes:

like this:

![image](https://user-images.githubusercontent.com/12274903/68355786-4f219f80-014b-11ea-8023-5770d7eb5176.png)

Reply:
Edited in https://github.com/CoinAlpha/hummingbot/pull/1037/commits/3242cdeeab3745fe063d4e069e96e35aee7733e5